invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/SharedOrMineFilter.js

```diff
@@ -0,0 +1,110 @@
+// This file is part of Invenio
+// Copyright (C) 2023 CERN.
+//
+// Invenio is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { withState } from "react-searchkit";
+import { Dropdown, Button } from "semantic-ui-react";
+
+class SharedOrMineFilterComponent extends Component {
```

Comment:
would there be a better place for this component? please check my comment in invenio_requests PR